build_tools/github_actions/runner/gcp/update_runner_version.py

```diff
@@ -44,6 +43,9 @@
 # introspect on its own source file location.
 TARGET_SCRIPT = "build_tools/github_actions/runner/gcp/create_templates.sh"
 
+# Typically we use "x86_64" but "x64" is used by Github runner.
+RUNNER_ARCHES = ["x64", "arm64"]
```

Comment:
nit: I think we usually pluralize the "arch" abbreviation of "architecture" as "archs" instead of "arches". Could just spell out "architectures" though.

Reply:
Done.